test/integration/count_test.lua

```diff
@@ -657,7 +641,7 @@ pgroup.test_count_no_map_reduce = function(g)
         },
     })
 
-    local router = g.cluster:server('router').net_box
+    local router = g.cluster.main_server.net_box
```

Comment:
To be honest, I'm against using `main_server` for a router because `main_server` is just a "first server in a cluster" and any reordering will break the assumption. Cartridge helper `:server('router')` finds the server by its name, so it's much stable

Reply:
Replaced by a function `helpers.get_router()` that returns `g.cluster:server('router')` for the cartridge.